### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by mahnouman**
Refactored code

### CODE REVIEW

**Comment by mahnouman** [comment]
Hello @dinis-cruz-gw @DinisCruz please merge this, thanks

File: stable-src/README.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
+# reverse-proxy-icap-docker (Release 2)
+
+[Source](https://github.com/k8-proxy/k8-reverse-proxy)
+
+Release 2 includes the completed/tested pieces of the Reverse Proxy project as a "reverse-proxy-icap-docker" to run inside a standard Ubuntu 18.04 (WSL and contianerized environements arent supported) server OVA image. The completed pieces of this project so far are:
```
Comment (DinisCruz) [inline stable-src/README.md:5]: Should be release 0.2

Comment (DinisCruz) [reply stable-src/README.md:5]: Next version needs to use Centos

**Review by DinisCruz — APPROVED** [review]
Great work

**Pull request merged**